backend/src/test/java/gov/cdc/usds/simplereport/db/repository/ResultRepositoryTest.java

```diff
@@ -115,6 +115,6 @@ void findAllByDiseaseSuccessful() {
         PATIENT, FACILITY, TestResult.POSITIVE, TestResult.NEGATIVE, TestResult.NEGATIVE, false);
 
     List<Result> results = _repo.findAllByDisease(diseaseService.covid());
-    assertEquals(2, results.size());
+    assertEquals(4, results.size());
```

Comment:
2 covid results for line 112 (TestEvent result and TestOrder result)
2 covid results for line 114 (TestEvent result and TestOrder result)